heroku/cli
# refactor(redis-v5): Move 'redis:stats-reset' to CLI

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001awyu1YAA/view

This converts `redis:stats-reset` to oclif and updates its associated tests.

### Testing

1) Run `./bin/run redis:stats-reset -a APP` with this branch pulled down with an app that has redis on it 
2) Verify the command resets the stats
3) Compare the results with prod cli command and make sure things look 👍
